test/behaviour/concept/serialization/json/JSONTest.java

```diff
@@ -31,17 +31,17 @@
         strict = true,
         plugin = "pretty",
         glue = "com.vaticle.typedb.client.test.behaviour",
-        features = "external/vaticle_typedb_behaviour/concept/serialization.feature",
+        features = "external/vaticle_typedb_behaviour/concept/serialization/json.feature",
         tags = "not @ignore and not @ignore-typedb"
 )
-public class SerializationTest extends BehaviourTest {
+public class JSONTest extends BehaviourTest {
```

Comment:
I think @flyingsilverfin 's suggestion was `JSONSerializationTest`, for maximum explicitness.

Reply:
It's already scoped under `serialization`.

The suggestion was to have either `serialization-json` or `serialization/json`. I went with the latter, as this naming follows other tests. E.g. we have `MatchTest` under `typeql`, not `TypeQLMatchTest`.